### PR TITLE
[14.0][UPD] product_multi_barcode: Added menu to see all barcodes. Up…

### PR DESCRIPTION
**Pull request by ilyasProgrammer**
…dated product variant easy edit form to see barcodes.

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
The view in product variant is to reflect the possible presence of multiple barcodes.

The menu addition is to allow an easy import of multi-barcodes. If the added dependency on 'stock' is an issue we can create another module just for the menu, but being a module of repo 'stock-logistics-barcode' thought it would not be too much of a problem. 

@OCA/logistics-maintainers let us know what you think :)

**Review by elvise — APPROVED** [review]
LGTM!

File: product_multi_barcode/__manifest__.py [hunk 1]
```diff
@@ -15,11 +15,12 @@
     "ADHOC SA, "
     "Odoo Community Association (OCA)",
     "category": "Product Management",
-    "depends": ["product"],
+    "depends": ["product", "stock"],
```
Comment (rousseldenis) [inline product_multi_barcode/__manifest__.py]: IMHO adding a new dependency should be avoided.

You can add a new module instead.

Comment (ilyasProgrammer) [reply product_multi_barcode/__manifest__.py]: ok

**Review by francesco-ooops — APPROVED** [review]
@rousseldenis ok, now this enhancement PR contains only multi-barcode lines in variant view

menu has been moved to #413 

good for merge? :)

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**